modules/authentication/pom.xml

```diff
@@ -29,6 +29,11 @@
     <name>${project.artifactId}</name>
 
     <dependencies>
```

Comment:
We require it for credentials decoding in DefaultAuthentication.

The previous version was using version 1.2 and I deleted it from my laptop and cannot find that dependency again.  So I used version 1.3.